.github/ISSUE_TEMPLATE/feature.yml

```diff
@@ -0,0 +1,36 @@
+name: Feature Request
```

Comment:
We don't have issues enabled now, do we want to?

Reply:
@ImTheKai Do we? I think it makes sense to enable them

Reply:
Then please enable them whoever has permissions :)